client/src/components/users/types.ts

```diff
@@ -39,6 +40,21 @@ export class UserCollection {
     return this.users.length;
   }
 
+  getVisibleUsers(filters: MsFilterLabels): Array<UserModel> {
```

Comment:
Be mindful of selection: selecting all will select hidden users, and selecting a user and then hiding them will not reset the selection. I think simply adding a function will not be enough.